pkg/cmd/pulumi/deployment_settings_config.go

```diff
@@ -617,23 +655,31 @@ func configureGitPassword(ctx context.Context, be backend.Backend,
 	return nil
 }
 
-func configureGitSSH(ctx context.Context, be backend.Backend, s backend.Stack,
-	sd *workspace.ProjectStackDeployment, gitSSHPrivateKeyPath string,
-) error {
+func configureGitSSH(d *deploymentSettingsCommandDependencies, gitSSHPrivateKeyPath string) error {
 	if gitSSHPrivateKeyPath == "" {
-		return errors.New("No SSH private key was provided")
+		configureMsg := "No SSH private key was provided, run `pulumi deployment settings " +
+			"configure` with the `--git-ssh-private-key` flag set"
+		configureMsg = colors.Highlight(configureMsg, "No SSH private key was provided", colors.Red+colors.Bold)
```

Comment:
```suggestion
		configureMsg = colors.Highlight(configureMsg, "No SSH private key was provided", colors.SpecError+colors.Bold)
```

Similar for other places I think. It would be nicer to use topical colors where possible, which makes it easier to unify colors throughout.

Reply:
done